unix/runner

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+"""
+Script to prepare JSON output for tidal sync servers from the list of hosts
+"""
+
+from __future__ import division
+
+import json
+
+from collections import defaultdict, namedtuple
+
+from ansible.executor.task_queue_manager import TaskQueueManager
+from ansible.inventory.manager import InventoryManager
+from ansible.parsing.dataloader import DataLoader
+from ansible.playbook.play import Play
+from ansible.plugins.callback import CallbackBase
+from ansible.vars.manager import VariableManager
+
+TOTAL_RESULTS = {'servers': []}
+
+def ram_allocated_gb(facts):
+    """Return total memory allocation in GB"""
+    return facts['ansible_memtotal_mb'] / 1024
+
+def ram_used_gb(facts):
+    """Return used memory in GB"""
+    return (facts['ansible_memtotal_mb'] - facts['ansible_memfree_mb']) / 1024
+
+def _size(key, mounts):
+    sizes = []
+    for item in mounts:
+        item = defaultdict(lambda: 0, item)
+        sizes.append(item[key])
+    return sum(sizes)
```

Comment:
If I understand this correctly, `mounts` is a list of hashes, and you need to add up all the values at `key` in each hash, using `0` as the default value if `key` is not in the hash.

If that's the case, then you can use the `get` method instead of creating a `defaultdict`:
```python
def _size(key, mounts):
    sizes = []
    for item in mounts:
        sizes.append(item.get(key, 0))
    return sum(sizes)
```

Also, I think this would be a good place for a [list comprehension](https://docs.python.org/2/tutorial/datastructures.html#list-comprehensions):
```python
def _size(key, mounts):
    return sum([item.get(key, 0) for item in mounts])
```

Reply:
Thanks a lot for the `get` method! I previously used something like `return sum(item[key] for item in mounts)` but didn't know how to deal with missing keys. Now I know, thanks!